toxcore/query.c

```diff
@@ -0,0 +1,190 @@
+/** query.c
+ *
+ * Makes requests for information using the DHT or Onion as approitate.
+ *
+ * This file should include a licence, but if you're reading this, it doesn't yet so everything in
+ * this file is donated to both tox, toktok, and you under a BSD or MIT licence at any option.
```

Comment:
Licensing doesn't work like that. This is licensed under GPL, do it properly. Same goes for the .h file.